### ISSUE TITLE
Feedback : WPML Support, Only Sync In Stock, Support Visibility Quick Edit

### ISSUE THREAD

**Issue by KjellBetlem** [facebookincubator/facebook-for-woocommerce#20]
First of all thanks for the great plugin. This is a really nice add-on to market Woocommerce shops. I have some minor feedback for improvement that I would like to share:

- When using Woocommerce with the WPML multilingual plugin, translated products are synced twice: both in the original language and in the translated language (See https://wpml.org/forums/topic/facebook-for-woocommerce-extension-products-displaying-twice/)

- It would be great if the plugin could offer an option to only sync products to Facebook that are currently in stock. 

- At the moment the plugin offers the option to manually choose which products to show on Facebook. It would be really convenient if it would be possible to edit this option for multiple products in one go, using quick edit mode    

Thanks!

**Comment by gchterry** [comment]
edit current sync
filter the wordpress as the language as u want, redo again product sync
i use polylang like this no problem for sync only 1 language product

**Comment by dmitridr** [comment]
Hello,

Given that we're dealing with a high volume of bugs, we unfortunately won't get to thes feature requests for a long time to come, even though we'd like to. However, we'd be happy to review any pull request to add this functionality, please consider sending us one!

**Comment by mengyingdu** [comment]
Hi, 
We included fixes for WPML in [v1.7.4](https://github.com/facebookincubator/facebook-for-woocommerce/releases/tag/v1.7.4). Can you upgrade and try it?

**Comment by dmitridr** [comment]
As Mengying said, we now support WPML >=3.2. The default behavior for the plugin is still to sync all the products, but the products which are not in the default language will be invisible to users seeing the shop.

**Comment by dmitridr** [comment]
Closing this, as we have addressed 2/3 of these issues, and likely will not address the out-of-stock one. We can follow up in #124 on that feature. 

The reason that would be tricky is there are merchants who may still want to keep out of stock items synced to facebook, in order to gain the benefit of insights from customers viewing out of stock products and the creation of custom audiences based on these events. The performance of Dynamic Ads also benefits from having these synced.
